csharp/src/Ice/Connection.cs

```diff
@@ -221,11 +238,11 @@ public void Close(ConnectionClose mode)
 
         /// <summary>Gets the ACM parameters.</summary>
         /// <returns>The ACM parameters.</returns>
-        public ACM GetACM()
+        public Acm GetAcm()
```

Comment:
Shouldn't we replace GetAcm/SetAcm with an Acm property?

Reply:
Fine with me, currently SetAcm accepts nullable values and use the default if a value is set to null, Instead of that the user can retrieve Acm before setting and then change the values he wants in case he only wants to change one of the values.

Reply:
Right... or I guess we could provide a Clone method similar to proxies but probably a bit overkill.

Reply:
Fixed